### PR TITLE
Feature project list

### PR DESCRIPTION
**Pull request by sachind-cuelogic**
Create project list fetch all data from database, It display only logged in users project who is working on that projects.

### CODE REVIEW

File: bms/bms_app/models.py [hunk 1]
```diff
@@ -29,10 +30,54 @@ def __str__(self):
...
     prod_user = models.ForeignKey(User, null=True)
     product = models.ForeignKey(ProductDetails, null=True)
     prod_user_role = models.ForeignKey(UserRole)
+    bug_data = models.ForeignKey('Bug_Details',null=True)
+
+    # @staticmethod
+    # def get_product_user_list(current_user):
+    #     product_user_list = ProductUser.objects.filter(prod_user_id=current_user).values('product__id', 'product__prod_version', 'product__prod_name', 'prod_user_role__role')
+    #     print product_user_list
+    #     return product_user_list
```
Comment (ananthu-nair707) [inline bms/bms_app/models.py]: Do you need this code?

File: bms/bms_app/models.py [hunk 2]
```diff
@@ -29,10 +30,54 @@ def __str__(self):
...
+    bug_assigned_to = models.ForeignKey(User, related_name='bug_assigned_to')
+    bug_file = models.FileField(upload_to='documents/%Y/%m/%d', null=True)
+
+    def __str__(self):
+        return self.title
+
+
+    # def count_bug():
+    #     no_of_bug = Bug_Details.objects.all().count()
+    #     print no_of_bug
```
Comment (ananthu-nair707) [inline bms/bms_app/models.py]: Required code?

File: bms/bms_app/models.py [hunk 3]
```diff
@@ -29,10 +30,54 @@ def __str__(self):
...
+    def __str__(self):
+        return self.prod_user_role
 
     def __str__(self):
         return u'{0}'.format(self.prod_user_role)
+
+
+class BugType(models.Model):
+    bug_name = models.CharField(max_length=50)
+
```
Comment (ananthu-nair707) [inline bms/bms_app/models.py]: Are bug types constants or will they change?

File: bms/bms_app/views.py [hunk 4]
```diff
@@ -115,8 +107,38 @@ def create_product(request):
...
+        # user_list = ProductUser.objects.raw("SELECT "
+        #                 "pu.id, pd.prod_name, pd.prod_version, ur.role, pd.id as product_id, bd.title "
+        #                 "FROM bms_app_productuser pu "
+        #                 "JOIN bms_app_productdetails pd ON pd.id = pu.product_id "
+        #                 "JOIN bms_app_userrole ur ON ur.id = pu.prod_user_role_id "
+        #                 "JOIN bms_app_bug_details bd ON bd.id = pu.bug_data_id "
+        #                 "where pu.prod_user_id = %s ", [current_user.id])
+
+        print user_list
+
```
Comment (ananthu-nair707) [inline bms/bms_app/views.py]: Commented code + print/debug statements..please remove.

**Review by ananthu-nair707 — CHANGES_REQUESTED** [review]

File: bms/bms_app/models.py (hunk 3 above)
Comment (sachind-cuelogic) [reply bms/bms_app/models.py]: These are constants

**Review by ananthu-nair707 — APPROVED** [review]

**Pull request merged**